suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -88,9 +88,13 @@ const finishNetworkTypeDiscoveryThunk = createThunk(
 );
 
 const getDetailsLevels = (coin: NetworkSymbol) => {
-    const network = getNetwork(coin);
+    const networkType = getNetworkType(coin);
     // For Cardano we need to fetch at least one tx otherwise it will not generate correctly new receive addresses (xpub instead of address)
-    if (network?.networkType === 'cardano') return { details: 'txs', pageSize: 1 } as const;
+    if (networkType === 'cardano') return { details: 'txs', pageSize: 1 } as const;
+
+    // For Ethereum we need to fetch token balances to be able to get their fiat rates and later display them in the UI.
+    if (networkType === 'ethereum') return { details: 'tokenBalances' } as const;
+
     return { details: 'tokens' } as const;
```

Comment:
What's difference between `tokens` and `tokenBalances`. Maybe we should put default fallback to `tokenBalances`?

Reply:
`token` return array of included account tokens metadata. `tokenBalances` return the same array, but also information how much balance the account have.

Btw. why are we fetching `tokens` for any network? Should not be enought to fetch `basic` by default? Then only `tokenBalances` for eth and `txs` for Cardano.

Reply:
You may try to set `basic` or `tokenBalances` instead.

Reply:
I am confident that for all the other coins other then ethereum like ones, the `basic` level is sufficient. Fixed with ba5c3b5f98.